### PR TITLE
Add links for latest release for each language

### PR DESCRIPTION
**Pull request by weshaggard**
PTAL @adrianhall @kaerm @loarabia @jongio @maggiepint 

fyi @Petermarcu @kurtzeborn 

If you want to see a preview of what this looks like https://weshaggard.github.io/azure-sdk/releases/latest/

### CODE REVIEW

File: _data/releases/latest/python-packages.csv [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Core,azure-core,1.0.0,core
+Identity,azure-identity,1.0.0,identity
+EventHubs,azure-eventhub,5.0.0b5,eventhub,azure-eventhubs
+EventHubs - CheckPoint,azure-eventhub-checkpointstoreblob,1.0.0b5,eventhub,azure-eventhubs-checkpointstoreblob
+EventHubs - CheckPoint AIO,azure-eventhub-checkpointstoreblob-aio,1.0.0b5,eventhub,azure-eventhubs-checkpointstoreblob-aio
+Key Vault - Certificates,azure-keyvault-certificates,4.0.0b5,keyvault
+Key Vault - Keys,azure-keyvault-keys,4.0.0,keyvault
+Key Vault - Secrets,azure-keyvault-secrets,4.0.0,keyvault
+Storage - Blobs,azure-storage-blob,12.0.0,storage
+Storage - File Shares,azure-storage-file-share,12.0.0b5,storage
```
Comment (kaerm) [inline _data/releases/latest/python-packages.csv]: 12.0.0 - it is no longer in preview

File: _data/releases/latest/js-packages.csv [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Service,Package,Version,RepoPath
+App Configuration,app-configuration,1.0.0-preview.7,appconfiguration
+Identity,identity,1.0.0,identity
+EventHubs,event-hubs,5.0.0-preview.6,eventhub
+Key Vault - Certificates,keyvault-certificates,4.0.0-preview.9,keyvault
+Key Vault - Keys,keyvault-keys,4.0.0,keyvault
+Key Vault - Secrets,keyvault-secrets,4.0.0,keyvault
+Storage - Blobs,storage-blob,12.0.0,storage
+Storage - File Shares,storage-file-share,12.0.0-preview.6,storage
```
Comment (kaerm) [inline _data/releases/latest/js-packages.csv]: storage-file-share,12.0.0 - no longer preview

File: _data/releases/latest/java-packages.csv [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+App Configuration,azure-data-appconfiguration,1.0.0-preview.6,appconfiguration
+Identity,azure-identity,1.0.0,identity
+EventHubs,azure-messaging-eventhubs,5.0.0-preview.5,eventhubs
+Key Vault - Certificates,azure-security-keyvault-certificates,4.0.0-preview.5,keyvault
+Key Vault - Keys,azure-security-keyvault-keys,4.0.0,keyvault
+Key Vault - Secrets,azure-security-keyvault-secrets,4.0.0,keyvault
+Storage - Blobs,azure-storage-blob,12.0.0,storage
+Storage - Blobs Batch,azure-storage-blob-batch,12.0.0,storage
+Storage - Blobs Cryptography,azure-storage-blob-cryptography,12.0.0,storage
+Storage - File Shares,azure-storage-file-share,12.0.0-preview.5,storage
```
Comment (kaerm) [inline _data/releases/latest/java-packages.csv]: 12.0.0 - no longer in preview

File: _data/releases/latest/dotnet-packages.csv [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+Identity,Azure.Identity,1.0.0,identity
+EventHubs,Azure.Messaging.EventHubs,5.0.0-preview.5,eventhub
+Key Vault - Certificates,Azure.Security.KeyVault.Certificates,4.0.0-preview.6,keyvault
+Key Vault - Keys, Azure.Security.KeyVault.Keys,4.0.0,keyvault
+Key Vault - Secrets,Azure.Security.KeyVault.Secrets,4.0.0,keyvault
+Storage - Blobs,Azure.Storage.Blobs,12.0.0,storage
+Storage - Blobs Batch,Azure.Storage.Blobs.Batch,12.0.0,storage
+Storage - Common,Azure.Storage.Common,12.0.0,storage
+Storage - DataLake,Azure.Storage.Files.DataLake,12.0.0-preview.6,storage
+Storage - File Shares,Azure.Storage.Files.Shares,12.0.0-preview.5,storage
```
Comment (kaerm) [inline _data/releases/latest/dotnet-packages.csv]: Azure.Storage.Files.Shares,12.0.0,storage - no longer in preview

File: _data/releases/latest/python-packages.csv (hunk 1 above)
Comment (weshaggard) [reply _data/releases/latest/python-packages.csv]: I'm actively working on a script that will pull the latest version and update these files. For now I just wanted to get the tables PR'ed so you guys can have a look.

Comment (weshaggard) [reply _data/releases/latest/python-packages.csv]: It has not yet shipped non-preview so we cannot update this table to have valid pointers until it has shipped.

File: _data/sidebars/releases_sidebar.yml [hunk 5]
```diff
@@ -4,7 +4,9 @@ entries:
   folders:
     - title: 2019 Releases
       folderitems: 
-        - title: November (Latest)
+        - title: Latest
```
Comment (adrianhall) [inline _data/sidebars/releases_sidebar.yml]: Wouldn't this be December (Latest) - or do we always want to link the latest to latest?

Comment (weshaggard) [reply _data/sidebars/releases_sidebar.yml]: I intend this to always be latest and when we want to do a snapshot we copy this and add the date but this one will always remain and always be the latest.

**Pull request merged**